### PR TITLE
Support deprecation in csv generation

### PR DESCRIPTION
**Pull request by danielwilms**
* Added deprecation to vss element
* Added deprecation to csv output
* Fixed missing UUID

Fixes: #33

### CODE REVIEW

File: model/constants.py [hunk 1]
```diff
@@ -101,6 +101,7 @@ class VSSType(Enum):
     SENSOR = "sensor"
     ACTUATOR = "actuator"
     ELEMENT = "element"
+    DEPRECATION = "deprecation"
```
Comment (danielwilms) [inline model/constants.py:104]: has to go

Comment (danielwilms) [reply model/constants.py]: Fixed, please check if it's now OK for you @SebastianSchildt, @gunnarx

**Comment by SebastianSchildt** [comment]
Deprecation fix looks fine for me, on the CSV I have to trust you :)

**Comment by danielwilms** [comment]
@gunnarx, so maybe we could merge it. then I can update the vss-tools submodule.

**Review by SebastianSchildt — APPROVED** [review]

**Comment by gunnarx** [comment]
Squashed and merged

**Pull request merged**